script/Arcueid/Day 11/11_00_ARC11_3.txt

```diff
@@ -1019,24 +1066,24 @@ Even here, like this―――hearing your heartbeat is driving me mad with passi
 [sha:7251e16545ede5b2abc9d540ea244b61464c648d]{
 -- Page 224, Offset 30149.
 -- とくん、と。
-Badum, badum.
+%{i}Badump.%{/i}
 }
 [sha:2269b360994fd0f095bd10ce5dbab90ca80c3274]{
 -- Page 224, Offset 30150.
 -- 抱きしめた腕から、重なり合った体から、
-Eminating from the chest I'm holding tight; from our overlaid bodies.
+Emanating from the arms I'm holding tight, from the body pressed against mine―――
 }
 [sha:6572b275af2dd55b185d917b205e084cd9151b1c]{
 -- Page 224, Offset 30151.
 -- アルクェイドの鼓動が、感じられる。
-I can feel the palpations of Arcueid's heart.
+I can feel the palpitations of Arcueid's heart.
 }
 [sha:ac759a73ff24392a45e511be302fc9fab31cbd22]{
 -- Page 225, Offset 30152.
 -- 「……ちがうよ、志貴。それは……ただこの瞬間だけ、わたしに、気が触れてる、だけだもの……」
-// ellipses are what I'm guessing are Arcueid's pausing because shiki is hugging
-// her so tight. I don't have the game on hand so I can't check atm.
-"You've got it wrong, Shiki. I'm only... making you...crazy for the moment, really..."
+"You've got it wrong, Shiki.
+ Those feelings for me are just...
+ nothing more than a passing craze..."
```

Comment:
Not a fan of "passing craze", makes it sound like a trend more than temporary madness.